js/libs/core.js

```diff
@@ -697,6 +701,54 @@ Fliplet.FormBuilder = (function() {
         this.value = '';
       };
 
+      component.methods._getDataSourceColumnValues = function() {
+        var $vm = this;
+        var id = this.dataSourceId;
+        var column = this.column;
+
+        Fliplet.Cache.get({
+          key: id + '-' + column,
+          expire: 60
+        }, function getColumnValues() {
+          // If there's no cache, return new values, i.e.
+          Fliplet.DataSources.connect(id).then(function(connection) {
+            connection.getIndex(column).then(function onSuccess(values) {
+              $vm.options = _.compact(values, function(value) {
+                if (value !== '') {
+                  return value.trim();
+                }
+              }).map(function(rawOption) {
+                if (rawOption) {
+                  rawOption = rawOption.trim();
+
+                  var regex = /<.*>$/g;
```

Comment:
If the data comes from the data source, why would it contain `<*>`?

Reply:
I had considered it would contain

Reply:
If one of the unique value from data source entries is `Bar <Foo>`, then `Bar <Foo>` is the value. There's no need to parse it further.